administrate/config/locales/administrate.fr.yml

```diff
@@ -0,0 +1,23 @@
+---
+fr:
+  administrate:
+    actions:
+      confirm: Êtes-vous sûr ?
+      destroy: Supprimer
+      edit: Modifier
+      show: Voir
```

Comment:
Actually, this translation file is misleading... I think we actually need the verb `to show`, because we use it alongside the resource name like `Show Product`.

Ideally, this translation would have interpolation, like: `show: Voir %{resource_name}`, but that will need to happen in a separate PR.